bancorprotocol/carbon-app
# [Strategy Card] Truncate number to 6 decimal if range < 1

fix #1273

When order range delta is < 1 we truncate number after 6 decimals.

## Test cases :
- Create a disposable range strategy with theses prices
- Hover graph in strategy card to see prices (price is after sdk approximation)

**Both below 0**:
- Form: min 0.000000001 & max 0.01
- Card: min 0.0₈99999 & max 0.009999

**Above & below 0**:
- Form: min 0.99999 & max 1.00001
- Card: min 0.999989 & max 1.000009

**Both above 0**:
- Form: min 10.0005 & max 11.0000001
- Card: min 10.000499 & max 11.00
